assets/js/components/Collection/Form.jsx

```diff
@@ -10,6 +10,13 @@ import Error from "../UI/Error";
 import Loading from "../UI/Loading";
 import { toastWrapper } from "../../services/helpers";
 import { useForm } from "react-hook-form";
+import UIFormFieldAndDisplay from "../UI/Form/Field.jsx";
```

Comment:
This change might have just been introduced, but the name of the component has changed to `UIFormField`.  To avoid confusion, mind updating all references?